### PR TITLE
Update README.md

### LINKED ISSUE #208: `track2gather_32.png` is outdated
This issue will make more sense after my refactor comes out.

### LINKED ISSUE #235: repo README.md not updated with latest landing page index.md
The 2 files are seperate and needs to be updated when the other changes.

Could potentially make README.md always reflect index.md changes.

### PR DESCRIPTION
**Pull request by frederickpek**
Repo README.md not updated with latest landing page index.md, lets update it.

Include also, updated track2gather_32.png file.

This PR fixes #208 and fixes #235.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-    * case contacts
-    * quarantine period
-* For the detailed documentation of this project, see
-  the **[Track2Gather Website](https://ay2122s1-cs2103-w14-2.github.io/tp/)**.
+* Track2Gather is an app that provides basic features for contact tracing personnel to track the following details for
+  up to a few thousand persons:
+  * personal information (limited to English names and Singaporean contact numbers and addresses)
+  * case number
+  * Stay-Home-Notice period
+  * next-of-kin information
```
Comment (jennibearduit) [inline README.md:11]: Is it possible to add the word "enforce" somewhere here?

**Review by jennibearduit — CHANGES_REQUESTED** [review]
LGTM, minor suggestion.

File: README.md (hunk 1 above)
Comment (frederickpek) [reply README.md:11]: This is a good suggestion! Perhaps we could bring this up again in Saturday's meeting.

I will leave this as it is becuse its only to reflect the landing page changes.

Comment (jennibearduit) [reply README.md:11]: Ok sure!

**Review by jennibearduit — APPROVED** [review]
LGTM!

**Pull request merged**